harness/determined/pytorch/dsat/_utils.py

```diff
@@ -97,6 +98,31 @@ def get_full_parser() -> argparse.ArgumentParser:
         default=_defaults.AUTOTUNING_ARG_DEFAULTS["search-range-factor"],
     )
 
+    # TODO: just using the paper's notation now, but should make align with existing Determined.
+    asha_subparser = subparsers.add_parser("asha", parents=[base_parser])
+    asha_subparser.add_argument(
+        "--max-binary-search-trials",
+        default=_defaults.AUTOTUNING_ARG_DEFAULTS["max-binary-search-trials"],
+    )
+    asha_subparser.add_argument(
+        "--min-binary-search-trials",
+        default=_defaults.AUTOTUNING_ARG_DEFAULTS["min-binary-search-trials"],
+    )
+    asha_subparser.add_argument(
+        "--asha-early-stopping",
+        default=_defaults.AUTOTUNING_ARG_DEFAULTS["asha-early-stopping"],
+    )
+    asha_subparser.add_argument(
+        "--divisor",
+        default=_defaults.AUTOTUNING_ARG_DEFAULTS["divisor"],
+    )
+    asha_subparser.add_argument(
+        "-srf",
+        "--search-range-factor",
+        type=float,
+        default=_defaults.AUTOTUNING_ARG_DEFAULTS["search-range-factor"],
+    )
```

Comment:
Could we update the `README` to show some examples of how to use this?

Reply:
Yeah, that's the next general task: documentation. Presumably replacing all READMEs with official docs.

Reply:
This is mostly just a nit btw. Not blocking or anything I just wanted to notate it so I remembered